types/react-helmet/v4/index.d.ts

```diff
@@ -41,8 +41,6 @@ declare namespace ReactHelmet {
         toString(): string;
         toComponent(): React.Component<any>;
     }
-
-    class HelmetComponent extends React.Component<HelmetProps> {}
 }
 
 export = ReactHelmet;
```

Comment:
ReactHelmet is now an ES module (has been since v4 actually) but fixing that is for another PR